README.md

```diff
@@ -98,154 +98,112 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
- - Fixed high CPU usage when idle before initializing the main window. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
- - Implemented Ctrl+W as a shortcut to close Peek. Thanks [@Physalis2](https://github.com/Physalis2)!
- - Solved an issue incorrectly detecting encoding when previewing code files.
- - Fixed background issues when peeking into HTML files after the WebView 2 upgrade.
+ - Improved icon and title showing for previewed files. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+ - Added QOI file type support. Thanks [@pedrolamas](https://github.com/pedrolamas)!
 
 ### PowerToys Run
 
- - Moved to WPF-UI and redesigned according to Fluent UX principles. Thanks [@niels9001](https://github.com/niels9001)!
- - Fixed an issue causing 3rd party plugins to not have their custom settings correctly initialized with default values. (This was a hotfix for 0.75) Thanks [@waaverecords](https://github.com/waaverecords)!
- - Fixed a crash in the VSCode plugin when the VSCode path had trailing backspaces. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
- - Fixed a crash when trying to load invalid image icons.
- - Fixed a crash in the Programs plugin when getting images for some .lnk files.
- - Fixed a rare startup initialization error and removed cold start operations that were no longer needed. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
- - Improved calculations for Windows File Time and Unix Epoch Time in the DateTime plugin. Thanks [@htcfreek](https://github.com/htcfreek)!
- - Fixed a crash when trying to get the icon for a link that pointed to no file.
- - Cleaned up code in the WindowWalker plugin improving the logic. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+ - Fixed results list UI element height for different maximum number of results value. (This was a hotfix for 0.76)
+ - Fixed icon extraction for .lnk files. (This was a hotfix for 0.76)
+ - Fixed search box UI glitch when FlowDirection is RightToLeft. (This was a hotfix for 0.76)
+ - Fixed theme setting. (This was a hotfix for 0.76)
+ - Fixed error reporting window UI issue. Thanks [@niels9001](https://github.com/niels9001)!
+ - UI improvements and ability to show/hide plugins overview panel. Thanks [@niels9001](https://github.com/niels9001)!
+ - Allow interaction with plugin hints. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+ - Switch to WPF-UI theme manager. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+ - Fixed issue causing 3rd party plugin's dependencies dll not being loaded properly. Thanks [@coreyH](https://github.com/CoreyHayward)!
+ - Added configurable font sizes. Thanks [@niels9001](https://github.com/niels9001)!
+ - Fix plugins preview selection style. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
```

Comment:
```suggestion
 - Changed the text color of plugin hints to improve the contrast when light theme is used. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
```

The fix was an improvement of the PR created for the **Allow interaction with plugin hints** since in 0.76 you can't select plugins. I would mention the text color change with light theme.

Reply:
Didn't you fixed the selection style for HC themes too?

Reply:
Yeah, but it's not relevant for end users since 0.76 didn't have selection.